package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ProjectUnifyMobile",
+  "version": "1.1.1",
+  "description": "ProjectUnify Mobile: An Ionic project",
+  "dependencies": {
+    "gulp": "^3.5.6",
+    "gulp-concat": "^2.2.0",
+    "gulp-minify-css": "^0.3.0",
+    "gulp-rename": "^1.2.0",
+    "gulp-sass": "^2.0.4",
+    "phantomjs": "^2.1.3"
```

Comment:
@tochman remove this line. Don't forget to remove the comma at the end of the previous one once you do ;)